README.md

```diff
@@ -4,7 +4,7 @@
 
 # Getting Started
 
-This repository contains a copilot sample that can be used with the the [Azure AI Studio preview](https://aka.ms/azureai/docs).
+This repository contains a copilot getting sample that can be used with the the [Azure AI Studio preview](https://aka.ms/azureai/docs). 
```

Comment:
This looks like a typo - either "copilot sample" or "copilot getting started sample".